src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int BALL_COUNT = 3;
+    private static Lottery lottery = new Lottery();
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < BALL_COUNT; i++) {
+            System.out.println(lottery.getRandomBall().toString());
```

Comment:
```suggestion
            System.out.println(lottery.getRandomBall());
```
this method will be called automatically

Reply:
Thanks, forgot about it)